### PR TITLE
further tune postgresql on slaves

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
I hope this will make test runs a little bit faster.

File: puppet/modules/slave/manifests/postgresql.pp [hunk 1]
```diff
@@ -26,4 +31,27 @@
   postgresql::server::config_entry { 'synchronous_commit':
```
Comment (ekohl) [inline puppet/modules/slave/manifests/postgresql.pp]: Should we rewrite this to a hash:

```puppet
$settings = {}
$settings.each |$setting, $value| {
  postgresql::server::config_entry { $setting:
    value => $value,
  }
}
```

File: puppet/modules/slave/manifests/postgresql.pp [hunk 2]
```diff
@@ -3,6 +3,11 @@
     cwd => '/',
   }
 
+  class { 'postgresql::globals':
+    manage_package_repo => true,
+    version             => '9.6',
```
Comment (ekohl) [inline puppet/modules/slave/manifests/postgresql.pp]: Why 9.6 in particular? Shouldn't we try with the OS versions since that's what we deploy on?

Comment (mmoll) [reply puppet/modules/slave/manifests/postgresql.pp]: That's 9.2 with CentOS 7, while later versions do have some performance improvements.

File: puppet/modules/slave/manifests/postgresql.pp (hunk 1 above)
Comment (mmoll) [reply puppet/modules/slave/manifests/postgresql.pp]: good point 👍

**Comment by mmoll** [comment]
A different location wouldn't be problem as everything is provisioned newly by Puppet. The worst case could be that the old version DB server is used, but then a failed start of the new server would show up in the Puppet logs every run, I presume.

**Comment by ekohl** [comment]
I wonder about server sockets or duplicate running postgres servers.

**Comment by mmoll** [comment]
As we don't have remote_execution or something similar to control things when updating, I removed the update to 9.6 and included only parameters that are compatible with 9.2 now.

**Pull request merged**